mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/helpers/automation.ts

```diff
@@ -0,0 +1,10 @@
+import { select } from '@wordpress/data';
+import { storeName } from '../store';
+
+export function automationHasEmails(): boolean {
+  const automation = select(storeName).getAutomation();
+  const emailSteps = Object.values(automation.steps).filter(
+    (step) => step.key === 'mailpoet:send-email',
+  );
+  return emailSteps.length > 0;
+}
```

Comment:
This seems like it could (should?) got to Redux selectors.

Reply:
:100: fbc7fec8c